examples/ann/annoy_nsmlib_example.ipynb

```diff
@@ -0,0 +1,725 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x7faeff5e7190>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "\n",
+    "x = np.arange(10)\n",
+    "x = (x*10) * 1681\n",
```

Comment:
It's not clear what's happening here. Shouldn't `x` be stored as the same time as the timing is computed?